crates/rolldown/tests/rolldown/function/export_mode/umd/complex_name_named/artifacts.snap

```diff
@@ -0,0 +1,24 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+---
+# Assets
+
+## main.js
+
+```js
+import assert from "node:assert"
```

Comment:
Would be better if this doesn't show in the snapshot. https://github.com/rolldown/rolldown/pull/2489/files#r1812325035